modules/telco-core-cluster-common-use-model-engineering-considerations.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="telco-core-cluster-common-use-model-engineering-considerations_{context}"]
+= Telco core cluster common use model engineering considerations
+
+. Cluster workloads are detailed in "Application workloads".
+. Worker nodes should run on either of the following CPUs:
+.. Intel 3rd Generation Xeon (IceLake) CPUs or better when supported by {product-title}, or CPUs with the silicon security bug (Spectre and similar) mitigations turned off.
+Skylake and older CPUs can experience 40% transaction performance drops when Spectre and similar mitigations are enabled.
+.. AMD EPYC Zen 4 CPUs (Genoa, Bergamo, or newer) or better when supported by {product-title}.
++
+[NOTE]
+====
+Power consumption evaluations for these CPUs are ongoing.
+You should evaluate CPU features such as per-pod power management to determine any potential impact on your requirements.
+====
+.. IRQ balancing is enabled on worker nodes.
+The `PerformanceProfile` CR sets `globallyDisableIrqLoadBalancing` to false.
+Guaranteed QoS pods are annotated to ensure isolation as described in "CPU partitioning and performance tuning".
+
+. All cluster nodes should have the following features:
+.. Hyper-Threading enabled
+.. x86_64 CPU architecture
+.. Stock (non-realtime) kernel enabled
+.. Nodes not configured for workload partitioning
+
+. The balance between power management and maximum performance varies between `MachineConfigPool` groups in the cluster.
+The following configurations should be consistent for all nodes in a `MachineConfigPool` group.
+
+. Cluster scaling: see "Scalability" for more information.
+.. Clusters should be able to scale to at least 120 nodes.
+
+. CPU partitioning is configured using a `PerformanceProfile` CR and is applied to nodes on a per `MachineConfigPool` basis.
+See "CPU partitioning and performance tuning" for additional considerations.
+. CPU requirements for the OpenShift platform depend on the configured feature set and application workload characteristics.
+For a cluster configured according to the reference configuration, running a simulated workload of 3k pods as created by kube-burner node-density test the following CPU requirements are validated:
+.. The minimum number of reserved CPUs for supervisor and worker nodes is 2 CPUs (4 hyper-threads) per NUMA node.
```

Comment:
Is _supervisor node_ a telco term?

Reply:
Yes, they orchestrate where Telco CNF workloads go in the telco network. Can be control plane or not.